Makefile

```diff
@@ -181,6 +199,19 @@ else
 bpf-bytecode:
 endif
 
+#
+# tctl role tester
+# Requires a recent version of Rust and Cargo installed (tested rustc >= 1.52.1 and cargo >= 1.52.0)
+#
+ifeq ("$(with_roletester)", "yes")
+.PHONY: roletester
+roletester:
+	cargo build --manifest-path=$(ROLETESTER_BUILDDIR) --release
```

Comment:
The Rust project directory (containing the `Cargo.toml` file) should have should contain a file named `rust-toolchain` containing a toolchain name of the form `stable-yyyy-mm-dd` with the date filled in with the release date of the version we want to use. This forces Rust to use a precisely picked toolchain that we know works and ensures that CI always runs with the same toolchain that is used in practice.